disnake/interactions/base.py

```diff
@@ -622,9 +622,12 @@ async def defer(self, *, ephemeral: bool = False) -> None:
 
         Parameters
         -----------
+        with_message: :class:`bool`
+            Indicates whether the response will be a message with thinking state (bot is thinking...)
+            This only applies for interactions of type :attr:`InteractionType.component`, and is `True` otherwise.
```

Comment:
```suggestion
            Indicates whether the response will be a message with thinking state (bot is thinking...).
            This only applies for interactions of type :attr:`InteractionType.component`, and is ``True`` otherwise.
```

Reply:
https://github.com/DisnakeDev/disnake/pull/248/commits/aa40b4d28bf279c8ba54f288e87c5ec84818f827